.github/workflows/build_doc.yml

```diff
@@ -90,14 +100,14 @@ jobs:
             #list impacted packages
             LIST_OF_PKGS=$(git diff --name-only HEAD^1 HEAD |cut -s -d/ -f1 |sort -u | xargs -I {} echo {} && ls -d {}/package_info 2>/dev/null  |cut -d/ -f1 |egrep -v Installation||true)
             if [ "$LIST_OF_PKGS" = "" ]; then
-              echo "::set-output name=DoxygenError::No package affected."
+              echo "DoxygenError=No package affected." >> $GITHUB_OUTPUT
               exit 1
             fi
             cd build_doc && make -j2 doc
             make -j2 doc_with_postprocessing 2>tmp.log
             if [ -s tmp.log ]; then
               content=`cat ./tmp.log`
-              echo "::set-output name=DoxygenError::$(cat tmp.log)"
+              echo "DoxygenError=$(cat tmp.log)" >> $GITHUB_OUTPUT
```

Comment:
That will not work, because the content of `tmp.log` is a multilines string. Have a look at the documentation: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings